src/DotNetWorker/Context/Features/DefaultModelBindingFeature.cs

```diff
@@ -22,7 +22,7 @@ public DefaultModelBindingFeature(IEnumerable<IConverter> converters)
 
         public object?[]? InputArguments => _parameterValues;
 
-        public object?[] BindFunctionInput(FunctionContext context)
+        public object?[] TryBindFunctionInput(FunctionContext context)
```

Comment:
The pattern for `Try...` methods is that they return bool and "return" via out params. If you want this to be a `Try` (which I think makes sense), can you swap that signature around?